Meadowlark-js/backends/meadowlark-aws-dynamodb-opensearch/src/BaseDynamoRepository.ts

```diff
@@ -120,9 +120,10 @@ export function foreignKeyConditions(resourceInfo: ResourceInfo, documentInfo: D
       isDescriptor: false,
     };
 
-    if (documentReference.isAssignableFrom) {
-      return conditionCheckFromAssignable(entityTypeInfo, documentReference.documentIdentity);
-    }
+    // TODO: Needs updating to RND-275 changes
```

Comment:
Does that mean it _will_ be done in RND-275? If not, please create a separate ticket. If we axe the Dynamo support then we can just close that ticket.